README.md

```diff
@@ -2,6 +2,37 @@
 
 ## Quick start
 
+### Package common_ui from github registry
+
+#### Installation
+
+This app install the [@sigrennesmetropole/cooperation_jn_common_ui](https://github.com/sigrennesmetropole/cooperation_jn_common_ui/pkgs/npm/cooperation_jn_common_ui) package.
+Although this package is public, you still need a Private Access Token (PAT) from github to download it.
```

Comment:
Do you know why we need this PAT? It's strange to have this kind of thing while all is public :)